content/articles/Design a Sign-up and Login GUI using JavaFX/index.md

```diff
@@ -0,0 +1,553 @@
+                    <font>
+                        <Font size="14.0" />
+                    </font>
+                </RadioButton>
+                <CheckBox fx:id="termsConditionsCheckbox" layoutX="160.0" layoutY="545.0" mnemonicParsing="false" selected="true" style="-fx-text-fill: #A9A9A9;" text="I have read the Terms and Conditions">
+                    <font>
+                        <Font name="SansSerif Regular" size="15.0" />
+                    </font>
+                </CheckBox>
+                <Label fx:id="invalidSignupCredentials" alignment="CENTER" contentDisplay="CENTER" layoutX="160.0" layoutY="580.0" prefHeight="16.0" prefWidth="300.0" textAlignment="CENTER" textFill="RED">
+                    <opaqueInsets>
+                        <Insets />
+                    </opaqueInsets>
+                </Label>
+                <Button fx:id="signUpButton" layoutX="230.0" layoutY="615.0" mnemonicParsing="false" prefHeight="33.0" prefWidth="130.0" style="-fx-background-color: #24a0ed; -fx-text-fill: white; -fx-border-radius: 5; -fx-border-width: 2; -fx-border-color: #24a0ed;" text="Sign-Up">
+                    <font>
+                        <Font size="15.0" />
+                    </font>
+                </Button>
+                <Button layoutX="200.0" layoutY="660.0" mnemonicParsing="false" style="-fx-background-color: transparent; -fx-text-fill: #A9A9A9;" text="Terms and Conditions" underline="true">
+                    <font>
+                        <Font size="15.0" />
+                    </font>
+                </Button>
+                <!-- The End of Signup form -->
+
+            </children>
+        </AnchorPane>
+    </right>
+    <!-- Ending of the left part of the Borderpane -->
+
+    <!-- Beginning of the bottom part of the Borderpane -->
+    <bottom>
+
+        <!-- Simple Copyright -->
+        <Label prefHeight="16.0" prefWidth="107.0" style="-fx-text-fill: #B9a9a9;" text="Shop © 2021" BorderPane.alignment="CENTER" />
+    </bottom>
+    <!-- Ending of the bottom part of the Borderpane -->
+
+</BorderPane>
+
+```
+
+> **NOTE**: Paths may vary depending on Project Name and with your machine.
+>
+> In case you add controls and other elements manually using code, make sure you import them into the project.
+
+### Run the Application
+
+Run the application as before using the `Shift + F10` shortcut.
+The results are as shown below:
+
+![Final Product](/engineering-education/design-a-sign-up-and-login-gui-using-javafx/Final-product.png "Final Product")
+
+You can find the source code and files in [this](https://github.com/RisoriTofa/A-JavaFX-Login-and-Sign-Up-Form) repository.
+
+### Conclusion
+
+JavaFX is a very stable User Interface for developers developing computer pieces of software.
+It is not only interactive but more efficient and secure since one only uses elements they need and the User Interface files are separated from the System files hence there is no intermixing which may bring confusion.
```

Comment:
> It is not only interactive but more efficient and secure since one only uses elements they need and the User Interface files are separated from the System files hence there is no intermixing which may bring confusion.

This sentence is too wordy and should be broken up into 2-3 sentences or so. I wasn't sure how to fix this without removing your original meaning. This is since I wasn't sure if I understood it fully as you intend the meaning to be. Therefore, I'm leaving this up to you.

Reply:
Done!

Reply:
It seems there was a misunderstanding, I'm sorry if I was not clear. I wanted you to not commit any changes yet because I'm working on some minor changes to this branch that I'm going to commit. It would have been easier for me if you waited for me to commit my changes but it's too late now.

Reply:
> 
> 
> Done!

It seems you edited the wrong sentence. I was talking about the second sentence in the conclusion

Reply:
Is it well?